curriculum/challenges/english/blocks/lab-pricing-plans-layout/68e45a2b4ab9b2f48b1cc6df.md

```diff
@@ -0,0 +1,419 @@
+---
+id: 68e45a2b4ab9b2f48b1cc6df
+title: Design a Pricing Plans Layout Page
+challengeType: 25
+dashedName: design-a-pricing-plans-layout-page
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories**
+
+1. Your page should have a `h1` element with the text `Pricing Plans`.
+2. Your page should have a `div` element with the class `pricing-container`.
+    - The `div` element should have a `display` property with the value of `flex`.
+    - The `div` element should have a `flex-wrap` property with the value of `wrap`.
```

Comment:
I think these two lines should be replaced by a single user story:
> The `.pricing-container` element should have a `display` property with the value of `flex` and a `flex-wrap` property with the value of `wrap`.

Reply:
those are so granular that are more for hints than user stories